### ISSUE TITLE
Not compatible with Angular 13.2

### ISSUE THREAD

**Issue by jomedev** [ngx-grpc/ngx-grpc#76]
I tried to add the library to an Angular 13.2.2 project:

`npm i -S @ngx-grpc/common @ngx-grpc/core @ngx-grpc/grpc-web-client @ngx-grpc/well-known-types google-protobuf grpc-web`

But that gave me the following error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: angular-project@0.0.0
npm ERR! Found: rxjs@7.4.0
npm ERR! node_modules/rxjs
npm ERR!   rxjs@"~7.4.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer rxjs@"^6.0.0" from @ngx-grpc/common@2.4.1
npm ERR! node_modules/@ngx-grpc/common
npm ERR!   @ngx-grpc/common@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by smnbbrv** [comment]
Hi, the lib should be definitely updated.

Could you try temporarily to use --legacy-peer-deps? I assume it should work normally

**Comment by jomedev** [comment]
Thank you for your quick response! It worked with --legacy-peer-deps, but it would be nice not to need it.
Should I leave the issue open until it is updated?

**Comment by smnbbrv** [comment]
Yes please

**Comment by smnbbrv** [comment]
The issue is fixed in v3.0.0, see [CHANGELOG](https://github.com/ngx-grpc/ngx-grpc/blob/master/CHANGELOG.md#300-2022-03-01) for details
